src/styles/blocks/grid.scss

```
@@ -0,0 +1,30 @@
+.container {
+  --column-count: 1;
+  margin: 50px auto;
+  display: grid;
+  grid-template-rows: repeat(2, 408px);
+  grid-template-columns: repeat(var(--column-count), 200px);
+
+  row-gap: 46px;
+  column-gap: 48px;
+  justify-content: center;
+  align-items: center;
+}
+
+@media (min-width: 488px) {
+  .container {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    --column-count: 4;
+  }
+}
```

You can rewrite like this
```suggestion
.container {
  --column-count: 1;
  margin: 50px auto;
  display: grid;
  grid-template-rows: repeat(2, 408px);
  grid-template-columns: repeat(var(--column-count), 200px);

  row-gap: 46px;
  column-gap: 48px;
  justify-content: center;
  align-items: center;
  
  @media (min-width: 488px) {
    --column-count: 2;
  }

  @media (min-width: 768px) {
    --column-count: 3;
  }

  @media (min-width: 1024px) {
    --column-count: 4;
  }
}
```